docs/plugins/ingest-useragent.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[ingest-useragent]]
+=== Ingest Useragent Processor Plugin
+
+The Useragent processor extracts details from the useragent string a browser sends with its web requests. 
```

Comment:
Hope I caught all of them in https://github.com/elastic/elasticsearch/pull/19074/commits/5f8fa2334ef528e65ac6d09f200171b867c01399. In the code, it should now always be "useragent" or "Useragent", but never UserAgent, while in the comments and docs it should always be "user agent".